main.go

```diff
@@ -168,6 +168,11 @@ func parsePullRequest(conf *config, action string, pr *github.PullRequestEvent)
 	log.Info("Pull request event with action: ", action)
 	var builds []buildOptions
 
+	// Do not run the integration tests if '*NO-INTEGRATION*' is found on a separate line in the commit body
+	if pr.Body != nil && strings.Contains(pr.Body, "*NO-INTEGRATION*") {
```

Comment:
`strings.Contains` doesn't accept wildcards. Just the string alone should be enough.

I would also consider naming it `NO-RUN-TESTS`, since this is not limited to integration tests; acceptance tests also will benefit from this.

Reply:
They weren't wildcards :p It was supposed to be a part of the marker.